db/seeds/test_provider_populator.rb

```diff
@@ -12,6 +12,7 @@ class TestProviderPopulator
     "Test firm for portal login" => [807, %w[9F805X:21 3X805Z:22]],
     "Ahern & Co." => [808, %w[9F808X:21 3X808Z:22]],
     "Groves & Co." => [989, %w[7F809X:23 4X809Z:24]],
+    "Sugarman & daughters" => [989, %w[6F809X:25 6X809Z:26]],
```

Comment:
May need to change 989 and make it unique from the other entries?

Reply:
done